ci/gpu/build.sh

```diff
@@ -60,7 +60,7 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=${MINOR_VERSION}" \
       "statsmodels" \
-      "xgboost====1.0.2dev.rapidsai0.13" \
+      "xgboost==1.0.2dev.rapidsai0.13" \
```

Comment:
I made changes to these files originally because of ucx-py 0.15 not being released but I found this in the process. I don't think those extra ='s should be there.